### ISSUE TITLE
[native_pdf_view] Flutter 2.5 compatibility

### ISSUE THREAD

**Issue by Knupper** [rbcprolabs/packages.flutter#217]
**Describe the bug**
With flutter 2.5 the Gesture Detector has changed and now some plugins needs a update.

**To Reproduce**
1. Create a flutter project with flutter 2.5.0
2. add a dependencie to `native_pdf_view: ^4.1.0`
3. build 
4. Build is failing with following message: 

```
../../.pub-cache/hosted/pub.dartlang.org/photo_view-0.11.1/lib/src/core/photo_view_gesture_detector.dart:106:29: Error: The argument type 'PointerEvent' can't be assigned to the parameter type 'PointerDownEvent'.
 - 'PointerEvent' is from 'package:flutter/src/gestures/events.dart' ('../../fvm/versions/2.5.0/packages/flutter/lib/src/gestures/events.dart').
 - 'PointerDownEvent' is from 'package:flutter/src/gestures/events.dart' ('../../fvm/versions/2.5.0/packages/flutter/lib/src/gestures/events.dart').
    super.addAllowedPointer(event);
```

**Expected behavior**
Build is working without any problems

**Additional context**
The plugin (photo_view) is already updated to version `0.12.0`, but not released, so it would be great, when you can make a new release with this version.

**Comment by kinex** [comment]
As a workaround put this in your pubspec.yaml (at least builds): 
```
dependency_overrides:
  photo_view: ^0.12.0
```

**Comment by MalekJabri** [comment]
Hi kinex,

I tried that workaround but I can a message that native_pdf_view required the version 0.11.0

Thanks,

**Comment by kinex** [comment]
@MalekJabri I think you have put `photo_view` under `dependencies`, you must put it under `dependency_overrides`.

**Comment by dcvieira** [comment]
@kinex Thanks, this solve the issue.

**Comment by firatcetiner** [comment]
In either way this package should be using the latest version of the `photo_view` library. @SergeShkurko Any update on publishing a new version to `pub.dev`?

**Comment by mirshahbazi** [comment]
for now until the `photo_view` update on pub.dev use this : 
```
dependency_overrides:
  photo_view:
    # custom photo_view with better gesture recognition
    git: git://github.com/robertodoering/photo_view.git
```
this work for me 💯

**Comment by Knupper** [comment]
New version is released `4.2.0` so i close this issue.
